x-pack/plugins/endpoint/public/applications/endpoint/store/policy_list/reducer.ts

```diff
@@ -30,6 +32,25 @@ export const policyListReducer: Reducer<PolicyListState, AppAction> = (
     };
   }
 
+  if (action.type === 'serverFailedToReturnPolicyListData') {
+    return {
+      ...state,
+      apiError: action.payload,
+      isLoading: false,
+    };
+  }
+
+  if (action.type === 'userShownPolicyListServerFailedMessage') {
+    // Make sure that the apiError currently stored is the one the user was shown
```

Comment:
Trying to understand the flow here. If an error is stored in state, the react view uses the toast services to show info about it, then dispatches this. If the error matches the one in state, we clear it.

What is the point of keeping track of that in state?

Reply:
I'm trying to protect against showing the same error multiple times - example: during a quick paginations. That being said, I just realized that `useEffect()` has the error set as the dependency, so it will not fire multiple times unless it changes. Will look at it a little closer and remove it is not needed (will change API fetching middleware/reducer to clear error on fetch)